packages/test/end-to-end-tests/src/test/fluidObjectHandle.spec.ts

```diff
@@ -36,41 +36,6 @@ const tests = (args: ICompatLocalTestObjectProvider) => {
         await args.opProcessingController.process();
     });
 
-    it("should generate the absolute path for ContainerRuntime correctly", () => {
```

Comment:
FYI: I'll need to rewrite (partial this tests) and add UTs for FluidRoutingContext & co - this is coming.